force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.js

```diff
@@ -0,0 +1,165 @@
+import END_DATE_FIELD from "@salesforce/schema/ServiceSchedule__c.ServiceScheduleEndDate__c";
+import NUMBER_OF_SESSIONS_FIELD from "@salesforce/schema/ServiceSchedule__c.NumberOfServiceSessions__c";
+import SERVICE_SECHEDULE_ENDS_FIELD from "@salesforce/schema/ServiceSchedule__c.ServiceScheduleEnds__c";
+
+import SERVICE_SCHEDULE_INFORMATION_LABEL from "@salesforce/label/c.Service_Schedule_Information";
+import DATE_TIME_LABEL from "@salesforce/label/c.Service_Schedule_Date_Time";
+
+const FIELD_SET_NAME = "ServiceScheduleInformation";
+const WEEKLY = "Weekly";
+const ONE_TIME = "One Time";
+const ON = "On";
+const AFTER = "After";
+const LARGE_SIZE = 12;
+const SMALL_SIZE = 6;
+
+export default class NewServiceSchedule extends LightningElement {
+    @api serviceId;
+    @api recordTypeId;
+
+    fields;
+    fieldSet;
+    objectApiName = SERVICE_SCHEDULE_OBJECT.objectApiName;
+    isWeekly = false;
+    isRecurring = false;
+    isEndsOn = false;
+    isEndsAfter = false;
+    isLoaded = false;
+    selectedFrequency;
+    selectedDaysOfWeek;
+    seriesEnds;
+    picklists;
+
+    labels = {
+        scheduleInformation: SERVICE_SCHEDULE_INFORMATION_LABEL,
+        dateTime: DATE_TIME_LABEL,
+    };
+
+    sizes = {
+        large: LARGE_SIZE,
+        small: SMALL_SIZE,
+    };
+
+    dateFields = {
+        start: FIRST_SESSION_START_FIELD.fieldApiName,
+        end: FIRST_SESSION_END_FIELD.fieldApiName,
+        seriesEndsOn: END_DATE_FIELD.fieldApiName,
+        numberOfSessions: NUMBER_OF_SESSIONS_FIELD.fieldApiName,
+    };
+
+    picklistFields = {
+        frequency: FREQUENCY_FIELD.fieldApiName,
+        daysOfWeek: DAYS_OF_WEEK_FIELD.fieldApiName,
+        seriesEnds: SERVICE_SECHEDULE_ENDS_FIELD.fieldApiName,
+    };
+
+    @wire(getFieldSet, {
+        objectName: SERVICE_SCHEDULE_OBJECT.objectApiName,
+        fieldSetName: FIELD_SET_NAME,
+    })
+    wiredFieldSet({ error, data }) {
+        if (data) {
+            this.setFieldSet(data);
+        } else if (error) {
+            console.log(JSON.stringify(error));
+        }
+    }
+
+    setFieldSet(data) {
+        const recurrenceFields = Object.values(this.dateFields);
+        recurrenceFields.push(Object.values(this.picklistFields));
+
+        this.fieldSet = data
+            .filter(obj => !recurrenceFields.includes(obj.apiName))
+            .map(obj => {
+                let field = { ...obj };
+                field.size =
+                    obj.apiName === NAME_FIELD.fieldApiName ? LARGE_SIZE : SMALL_SIZE;
+                return field;
+            });
+    }
+
+    @wire(getObjectInfo, { objectApiName: SERVICE_SCHEDULE_OBJECT })
+    wiredObject(result) {
+        if (!result) {
+            return;
+        }
+
+        if (result.data) {
+            this.recordTypeId = this.recordTypeId
+                ? this.recordTypeId
+                : result.data.defaultRecordTypeId;
```

Comment:
the default doesn't come through via the `@api` above? only if it's set explicitly?

Reply:
Added a TODO to test with record types and check the behavior.